tests/test_workflow_introspection.py

```diff
@@ -856,3 +881,78 @@ async def child_workflow(id: str) -> str:
         res2 = await handle.get_result()
 
     assert res1 == res2
+
+
+def test_list_transaction(
+    dbos: DBOS, sys_db: SystemDatabase, app_db: ApplicationDatabase
+) -> None:
+
+    @DBOS.workflow()
+    def simple_workflow() -> None:
+        transactionOne()
+        stepTwo()
+        DBOS.sleep(1)
+        return
+
+    @DBOS.step()
```

Comment:
Those decorators need to be transactions.
```suggestion
    @DBOS.transaction()
```